### ISSUE TITLE
booleanWeekendsColor does not seem to work for mini calendar?

### ISSUE THREAD

**Issue by chuckedfromspace** [YU000jp/logseq-plugin-show-weekday-and-week-number#105]
Thank you very much for the nice plugin!

It seems that disabling `booleanWeekendsColor` in the settings menu doesn't disable the coloring of weekend days in the mini calendar above the title. Is it the intended behavior? Is there a way to change their colors in the mini calendar as well?
![image](https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/assets/43315257/e0cca4ca-bbf9-4299-8e75-935b767958b5)
![image](https://github.com/YU000jp/logseq-plugin-show-weekday-and-week-number/assets/43315257/eb8668d3-816a-4a20-a2c8-aa17dd429262)

Also, is it possible to expose more styling options for the mini calendar in the settings menu? Such as the border color for the current day (green). I would like it to be less prominent. Since the day spans are generated programmatically, it's not straightforward to use custom css to style it.

**Comment by chuckedfromspace** [comment]
It seems after refreshing Logseq the colors for weekend days are now correctly disabled in the mini calendar. Would still be nice to be able to set the highlight colors by the users though.

**Comment by YU000jp** [comment]
It was a bug that it was not reflected immediately.

Do you want to set colors for each day of the week?

**Comment by chuckedfromspace** [comment]
> It was a bug that it was not reflected immediately.
> 
> Do you want to set colors for each day of the week?

No, I would only like to change the border color of the selected day. Currently it's green if the opened journal page is today. The border is orange (?) if it is a day in the past or in the future. It would be nice to have them as "accent color" in the settings for users to set their colors of choice.
